mmdet3d/core/visualizer/show_result.py

```diff
@@ -89,6 +90,7 @@ def show_result(points,
         filename (str): Filename of the current frame.
         show (bool): Visualize the results online. Defaults to False.
```

Comment:
Please update the docstring for `show`, `snapshot`, and `pred_labels` with `optional` sign after the variable type.

For example:
`show (bool, optional): ... ...`.